scripts/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+services:
+  mongodb:
+    # most recent and stable version off mongodb
+    image: "mongo:4"
+    restart: always
+    ports:
+      # passthrough mongodb listen port to host for debugging
+      - "127.0.0.1:27017:27017"
+    volumes:
+      # store database data for a later restart
+      - type: volume
+        source: mongodb-data
+        target: "/data/db"
+    # setup authentication
+    #environment:
```

Comment:
```suggestion
    # environment:
```

Reply:
I have an "unspoken" indentation style for comments in configuration files. :smile: 
When I comment out configuration code, I only add a comment symbol (like `#` or `//`) to the line and no whitespace.
You can later distinguish the "real" configuration and the comment for the configuration.

If you say, that's not well indented, then we need to update the lines below and the configuration at `application/docker-compose.yml`, too. :wink:

Reply:
Ok... I guess the problem in this particular case is that it's not entirely clear that this is an earlier "part of code," so to speak. Initially, I had read this as "setup authentication environment:" (which, itself, would have been strange to put into two lines, as well, but "so be it").

Could we, perhaps, add a colon to the "setup authentication" part? Then, it would be a bit more clear that the "explainer comment" ends in line 16...

Reply:
Done in [`131e65f` (#123)](https://github.com/wuespace/telestion-project-daedalus2/pull/123/commits/131e65f7be10f58bdd631ec6ba5844eceea5eb9f). :wink: